Week1 [STEP 2] 코비

devKobe24 opened the pull request:
@Gundy93

STEP 1 PR 보내드립니다 건디 :)

 - The requested URL returned error: 403
 - 원격 저장소 url 변경하기
 - git 커밋 되돌리기
 - 원격 저장소 리스트 가져오기
 - Personal Access Token 설정하기
 - git commit amend
 - reset과 revert의 차이점 ---

 ## 해결이 되지 않은 점
 ### 주제
 - 터미널 내부에서 `#`을 사용하면 그 뒤에 내용은 무시된다고 써져있는데 PR 예시에는 `## 고민했던점` 이렇게 명시되어있었습니다.
 - 그렇다면 저 '고민했던점' 이라는 단어도 무시되는게 아닌가 궁굼합니다.

 ---

 ## 조언을 얻고 싶은 부분
 ### 주제
 - 어떤 상황에서 commit 내부에 백틱(`)을 사용하는지 궁굼합니다.

Commit:
Week1 [STEP 1] 코비

@Gundy93

STEP 1 PR 보내드립니다 건디 :)

 - The requested URL returned error: 403
 - 원격 저장소 url 변경하기
 - git 커밋 되돌리기
 - 원격 저장소 리스트 가져오기
 - Personal Access Token 설정하기
 - git commit amend
 - reset과 revert의 차이점
 ---

 ## 해결이 되지 않은 점
 ### 주제
 - 터미널 내부에서 `#`을 사용하면 그 뒤에 내용은 무시된다고 써져있는데 PR 예시에는 `## 고민했던점` 이렇게 명시되어있었습니다.
 - 그렇다면 저 '고민했던점' 이라는 단어도 무시되는게 아닌가 궁굼합니다.

 ---

 ## 조언을 얻고 싶은 부분
 ### 주제
 - 어떤 상황에서 commit 내부에 백틱(`)을 사용하는지 궁굼합니다.

Commit:
Week1 [STEP 2] 코비

@Gundy93

드디어 두 번째 스탭에 왔습니다 건디 !! :)

- 별을 그리고 막대를 그리는데 이중 for 문이 아닌 다른 방법이 없나 고민해봤습니다.

---

- 코드를 조금 더 깔끔하게 짜고 싶었습니다.
- 그런데 다른 방법이 생각이 나지 않았습니다.
- 건디라면 어떤 방법으로 짜셨을지 궁굼합니다.

---

- 저는 계속 이곳 저곳에 프린트를 찍어보면서 시도해보았습니다.
- 그런데 이렇게 문제를 푸는게 맞는것인지 의문이 들었습니다.
- 혼자 고민하고 한번에 풀어야 맞는게 아닌지 생각이 드는데 건디의 생각은 어떠신지 궁굼합니다.

Gundy93 reviewed (commented):
안녕하세요 코비님! 이번 PR도 이름이 STEP1이라서 순간 당황했습니다 ㅎㅎ
내용을 보니 STEP2 PR임을 알 수 있었습니다 😃

> 터미널 내부에서 #을 사용하면 그 뒤에 내용은 무시된다고 써져있는데 PR 예시에는 ## 고민했던점 이렇게 명시되어있었습니다.
> 그렇다면 저 '고민했던점' 이라는 단어도 무시되는게 아닌가 궁굼합니다.

우선 PR의 경우 깃허브에서 작성하기 때문에 터미널과는 무관한 것 같은데, 혹시 PR도 터미널에서 작성하신 것일까요? 이 부분은 디스코드로 다시 물어봐주시면 좋겠습니다 😃

commit의 경우라면 아래 사진과 같이 `""` 내부에 들어가는 `#`은 아무 문제가 생기지 않습니다.

<img width="448" alt="스크린샷 2023-02-25 오전 9 36 14" src="https://user-images.githubusercontent.com/106914201/221325716-1090ff80-fe9c-4068-9e3e-e2d44a5e55d4.png">

> 어떤 상황에서 commit 내부에 백틱(`)을 사용하는지 궁굼합니다.

커밋 메시지에 백틱을 사용하는 경우는 보통 코드에서 사용한 변수, 함수, 타입 등을 메시지에 적을 때 사용합니다. 가령 "feat: 아이스크림 몸통을 그리는 `iceCream` 함수 작성" 등과 같이요! 😃

이번 스텝에서 구현해야하는 순서도가 누락된 것 같습니다!
수정과 함께 순서도를 추가해주시면 될 것 같아요 👍

궁금하신 사항은 언제나 디스코드 DM 주시면 되겠습니다 😃

devKobe24 committed:
Merge branch 'yagom-academy:ss_7_kobe24' into ss_7_kobe24

Commit:
Week1 [STEP 2] 코비

Commit:
Revert "Week1 [STEP 1] 코비"

This reverts commit 2ff1973a6c510514d0a9fd9fe0a72d154b3280f4.

Week1 [STEP 2] 커밋 메세지 미작성으로 인한 revert

Commit:
Revert "Week1 [STEP 2] 코비"

This reverts commit eac13ad027da75d8d0a74896853894260ed180de.

Commit:
Week1 [STEP 2] Kobe

@Gundy93
우여곡절이 많았던 스탭 2의 PR 보내드립니다 건디 :)

 ## 고민했던 점
 ### git을 어떻게 더 잘 쓸 수 있을까?
 - 깃에 대한 공부를 더 해야겠다는 생각에 책 한권을 샀습니다.

---

 ## 해결이 되지 않은 점
 ### reset과 revert
 - reset과 revert의 차이점은 알았지만 직접 사용해보니 제 생각대로 되지 않았습니다.
 - 그래서 건디가 말씀해주신 해결 방법으로 해결해서 다행히도 PR을 보낼 수 있게 되었습니다.

---

 ## 조언을 얻고 싶은 부분
 ### 깃을 어떻게 공부해야 하는가.
 - 건디는 어떻게 깃을 공부하셨나요?
 - 저는 소스트리를 이용해서 그런지 터미널을 이용한 깃 커맨드는 너무 어렵네요

Commit:
Week1 [STEP 2 Kobe]
 @Gundy93
 건디 :) 수정사항 반영하여 다시 PR 보냅니다.

 ## 고민했던 점
 ### 아이스크림 몸통이 캠프 사이트에서 봤을때는 굉장히 사이가 멀어보였다.
 - print 함수 내부에서 줄바꿈을 해주고 있기 때문에 다시 줄바꿈 '\n'을 변수에 넣어줄 경우
   아이스크림 몸통 사이가 더 크게 별어진다.

 ---

 ## 해결이 되지 않은 점
 ### 캠프 사이트의 예시에 나온 캡쳐본과 조금 다른듯한 나의 출력물
 - print 함수 내부에서 줄바꿈을 해주고 있다는 건디의 어드바이스를 듣고 변수에 따로 '\n' 넣어줬던 코드를 뺏습니다.
   모양은 정말 이쁘게 잘 나오지만 캠프 사이트 예시의 캡쳐본과 조금 다른 출력물 같아서 맞게 코드를 짠건지 잘 모르겠습니다.

---

 ## 조언을 얻고 싶은 부분
 ### 개발 공부에 대한 부분
 - 저는 개발 공부를 할 때 주로 블로그를 참고하고 블로그 글을 읽고 제 블로그에 따로 정리하는 편 입니다.
   건디는 어떤식으로 공부를 하고 계시는지 궁굼합니다.
